.github/workflows/tokamak-public-nightly.yml

```diff
@@ -6,11 +6,371 @@ on:
+      ec2-instance-id-4: ${{ needs.start-runner-4.outputs.ec2-instance-id }}
+      ec2-instance-id-5: ${{ needs.start-runner-5.outputs.ec2-instance-id }}
+      ec2-instance-id-6: ${{ needs.start-runner-6.outputs.ec2-instance-id }}
+      ec2-instance-id-7: ${{ needs.start-runner-7.outputs.ec2-instance-id }}
+      ec2-instance-id-8: ${{ needs.start-runner-8.outputs.ec2-instance-id }}
+      ec2-instance-id-9: ${{ needs.start-runner-9.outputs.ec2-instance-id }}
+      ec2-instance-id-10: ${{ needs.start-runner-10.outputs.ec2-instance-id }}
+    steps:
+      - name: Runners status
+        run: |
+          echo "EC2 instances id:"
+          echo ${{ needs.start-runner-0.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-1.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-2.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-3.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-4.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-5.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-6.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-7.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-8.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-9.outputs.ec2-instance-id }}
+          echo ${{ needs.start-runner-10.outputs.ec2-instance-id }}
+          echo "Runner labels:"
+          echo ${{ needs.start-runner-0.outputs.label }}
+          echo ${{ needs.start-runner-1.outputs.label }}
+          echo ${{ needs.start-runner-2.outputs.label }}
+          echo ${{ needs.start-runner-3.outputs.label }}
+          echo ${{ needs.start-runner-4.outputs.label }}
+          echo ${{ needs.start-runner-5.outputs.label }}
+          echo ${{ needs.start-runner-6.outputs.label }}
+          echo ${{ needs.start-runner-7.outputs.label }}
+          echo ${{ needs.start-runner-8.outputs.label }}
+          echo ${{ needs.start-runner-9.outputs.label }}
+          echo ${{ needs.start-runner-10.outputs.label }}
+          echo "Finished setup arm64 runners"
+
+  start-runner-0:
+    name: Start EC2 Runner 0
+    runs-on: ubuntu-latest
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Start EC2 runner
+        id: start-ec2-runner
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: start
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          ec2-image-id: ami-01bd9dc38789ab007
+          ec2-instance-type: t4g.2xlarge
+          subnet-id: subnet-013cf87174bb21c25
```

Comment:
How about changing to default subnet? If you change subnet, we can delete all subnet and nat gateway!
I successed using default subnet https://ap-northeast-2.console.aws.amazon.com/vpcconsole/home?region=ap-northeast-2#SubnetDetails:subnetId=subnet-4aa53021.

Reply:
I updated. I thought that you want to setup on a private network ^^

Reply:
Oh, I'm sorry. I told you so... I know you work for this job really hard. I'm sorry my changes.

Reply:
Kk, never mind Steven ^^
Thank you very much!

Reply:
Thank you very much!!!
I removed the subnets and routing tables and nat gateway. 🙇🏼‍♂️